apps/laboratory/next.config.mjs

```diff
@@ -1,13 +1,18 @@
 /** @type {import('next').NextConfig} */
-// Keep in-sync with https://docs.walletconnect.com/advanced/security/content-security-policy
+const SHAKRA_UI = `'sha256-e7MRMmTzLsLQvIy1iizO1lXf7VWYoQ6ysj5fuUzvRwE='`
+/*
+ * Keep in-sync with https://docs.walletconnect.com/advanced/security/content-security-policy
```

Comment:
what happens if there's a mismatch

Reply:
We risk having docs that aren't accurate or CSP requirements that aren't tested